OmniKitUI/ViewControllers/InsertCannulaSetupViewController.swift

```diff
@@ -61,6 +59,7 @@ class InsertCannulaSetupViewController: SetupTableViewController {
         case initial
         case startingInsertion
         case inserting(finishTime: CFTimeInterval)
+        case finish
```

Comment:
Loop has always used `ready` for the final state in all the pod setup views (Pairing, Insert Cannula & Replace) and this does seem to succinctly and consistently describe this state, so I believe that continuing to use `ready` here for this seems reasonable. Perhaps it's just a matter to find a better name for the new state if there is some cognitive dissonance, even though I'm not actually sure what might be causing this when using `finish` for a new earlier state. I could more easily understand if the name and button label were `finished` and "Finished" which imply the current task has already been completed instead of `finish` and "Finish" which imply that action is needed to complete the current task (of inserting the cannula) which would seem to make sense.

Nevertheless, I could go with something like `needsInsertionCheck` for this new state with a button label of "Check Insertion" as it sort of matches the name of the routine that will be used to do the confirmation, checkCannulaInsertionFinished() even though this really isn't what is happening.